lib/nylas/resources/drafts.rb

```diff
@@ -87,5 +87,20 @@ def destroy(identifier:, draft_id:)
 
       [true, request_id]
     end
+    
+    # Send an draft.
+    #
+    # @param identifier [String] Grant ID or email account from which to delete an object.
+    # @param draft_id [String] The id of the draft to delete.
```

Comment:
nit: can you update these doc lines?

```suggestion
    # @param identifier [String] Grant ID or email account from which to send the draft.
    # @param draft_id [String] The id of the draft to send.
```